storybook/main.js

```diff
@@ -9,14 +9,11 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
-import rollupJson from '@rollup/plugin-json';
-import { mergeConfigs } from '@web/dev-server';
-import { fromRollup } from '@web/dev-server-rollup';
-import rollupCommonjs from '@rollup/plugin-commonjs';
-import { watchSWC } from '../web-test-runner.utils.js';
 
-/** @type { import('storybook-builder-wds').StorybookConfigWds } */
-const config = {
+import { merge } from 'webpack-merge';
+
+/** @type { import('@storybook/web-components-webpack5').StorybookConfig } */
+export default {
```

Comment:
A personal nit: how do you feel about including the `disableWhatsNewNotifications: true` setting in the `core` settings (like we do in Spectrum CSS)? Also: what if we set `disableTelemetry: true`?